Documents/Command/ConfigureAgentLogsUploadToCloudwatch/README.md

```diff
@@ -28,12 +28,20 @@ On setting this to true, SSM Agent will start logging to CloudWatchLogs of the A
 
 (Optional) Specify the log group name for logging in CloudWatchLogs. The log group would be created if not already present. If not specified, the logs would be present in the default log group 'SSMAgentLogs'.
 
+### EnableShare
+
+(Optional) Enable sharing of logs with CloudWatch of another AWS Account. The ShareDestination parameter needs to be passed if true.
+
+### ShareDestination
+
+(Optional) Destination in the format 'IAMUserAccessKeyID::IAMUserSecretAccessKey::DestinationLogGroup::DestinationLogStream'. The IAM User should have access to DescribeLogStreams, PutLogEvents on the destination CloudWatchLogs resource. The log group and stream should be present. Parameter will be ignored if EnableShare is false.
```

Comment:
Can we use Parameter Store and a SecureString parameter to pass this information?
It is not a best practice to have secrets in clear text - currently, the ShareDestination parameter is not an encrypted parameter and will be visible in the console.